Plaid API Endpoints Collection.json

```diff
@@ -6610,7 +6685,7 @@
 								"header": [],
 								"body": {
 									"mode": "raw",
-									"raw": "{  \n   \"client_id\": \"7f57eb3d2a9j6480121fx361\",\n   \"secret\": \"79g03eoofwl8240v776r2h667442119\",\n   \"access_token\": \"access-sandbox-71e02f71-0960-4a27-abd2-5631e04f2175\",\n   \"account_id\": \"3gE5gnRzNyfXpBK5wEEKcymJ5albGVUqg77gr\",\n   \"type\": \"credit\",\n   \"network\": \"ach\",\n   \"amount\": \"12.34\",\n   \"ach_class\": \"ppd\",\n   \"user\": {\n     \"legal_name\": \"FirstName LastName\",\n     \"email_address\": \"foobar@email.com\",\n     \"phone_number\": \"123-456-7890\",\n     \"address\": {\n       \"street\": \"123 Main St.\",\n       \"city\": \"San Francisco\",\n       \"region\": \"CA,\n       \"postal_code\": \"94053\",\n       \"country\": \"US\"\n     }\n   },\n   \"device\": {\n     \"ip_address\": \"198.30.2.2\",\n     \"user_agent\": \"Mozilla/5.0 (iPhone; CPU iPhone OS 13_5_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.1 Mobile/15E148 Safari/604.1\"\n   },\n   \"origination_account_id\": \"1111-1111-1111-1111\"\n }"
```

Comment:
If that's an actual secret we care about, make sure to rotate it out.

Reply:
When I tried to send the corresponding client_id in a request, I got an error that it was malformatted, so I'm pretty sure these are just nonsense strings that look vaguely like client ids and secrets rather than anyone's actual client id and secret. but good catch.